Tasks/2 - Prompt/promt.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+	<head></head>
+	<body>
+		<script>
+		let cost = parseFloat(prompt("How much does a loaf of bread cost?"));
+		let number = parseFloat(prompt("How many loaves do you want?"));
+		let result = cost * number;
+		alert ("The total cost is $" + " " + result);
```

Comment:
Remove unnessesary space symbol after `alert`

Also try to rewrite expression `"The total cost is $" + " " + result` with [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) technique

Reply:
have done